test/jdk/javax/swing/JTable/8236907/LastVisibleRow.java

```diff
@@ -195,19 +169,13 @@ public void run() {
                 Point point = new Point(cellRect.x, cellRect.y);
                 SwingUtilities.convertPointToScreen(point, table);
 
-                captureRect[0] = new Rectangle(point.x, point.y, table.getColumnCount() * cellRect.width, cellRect.height);
+                captureRect[0] = new Rectangle(point.x+5, point.y+2, table.getColumnCount() * cellRect.width - 10, cellRect.height-2);
```

Comment:
Please adjust line lengths. Some of the lines contain more than 80 characters.

Reply:
Updated.